proto/ignition/msgs/oriented_3d_box.proto

```diff
@@ -0,0 +1,45 @@
+
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "Oriented3DBoxProtos";
+
+/// \ingroup ignition.msgs
+/// \interface Oriented3DBox
+/// \brief A 3D oreinted bounding box
+
+import "ignition/msgs/header.proto";
+import "ignition/msgs/vector3d.proto";
+import "ignition/msgs/quaternion.proto";
+
+message Oriented3DBox
+{
+  /// \brief Optional header data
+  Header header = 1;
+
+  /// \brief Center of the bounding box
+  Vector3d center = 2;
+
+  /// \brief Orientation of the bounding box
+  Quaternion orientation = 3;
```

Comment:
what coordinate frame is this in? Should we add a note about this in the documentation, or can this be application-specific, which means we should leave the documentation as-is? cc @chapulina